src/SFA.DAS.AANHub.Application.UnitTests/CalendarEvents/Queries/GetCalendarEventsQueryValidatorTests.cs

```diff
@@ -36,20 +36,23 @@ public async Task ValidateMemberId_NotActiveMemberId_FailsValidation(Member memb
         var calendarEvents = (List<CalendarEventSummary>)null!;
         var calendarEventsReadRepositoryMock = new Mock<ICalendarEventsReadRepository>();
 
-        calendarEventsReadRepositoryMock.Setup(c => c.GetCalendarEvents(new GetCalendarEventsOptions
-        {
-            MemberId = member.Id,
-            FromDate = fromDate,
-            ToDate = toDate,
-            EventFormats = new List<EventFormat>(),
-            CalendarIds = new List<int>(),
-            RegionIds = new List<int>(),
-            Page = 1,
-            PageSize = 5
-        }, It.IsAny<CancellationToken>()))
+        calendarEventsReadRepositoryMock
+            .Setup(c => c.GetCalendarEvents(new GetCalendarEventsOptions
+            {
+                MemberId = member.Id,
+                FromDate = fromDate,
+                ToDate = toDate,
+                EventFormats = new List<EventFormat>(),
+                CalendarIds = new List<int>(),
+                RegionIds = new List<int>(),
+                Page = 1,
+                PageSize = 5
```

Comment:
probably not important, but our default page size in admin is 10

Reply:
Something for you to pick up then :-)